README.md

```diff
@@ -2,6 +2,14 @@
 
 An operator for Samba as a service on PVCs in kubernetes.
 
+## ⚠️ Maintenance Status ⚠️
+
+Maintainers and contributors wanted!
+
+We wish to thank everyone who has shown interest in using and/or contributing to the samba-operator! However, we feel it important to share the fact that samba-operator is now minimally maintained. This means that the current team tries to keep up with basic "housekeeping" tasks but has little to no time for working on features or even some non-critial bug fixes. This is at least partly due to changing priorities for our employer - and since most of us work on this project as $DAYJOB task we don't have as much time to spend on this operator.
```

Comment:
Missing 'is':
"...we feel it **is** important..."

Reply:
Serves me right for trying to do this quickly using the GH UI, now I need to figure out how to edit a commit. :-)

Reply:
@phlogistonjohn  qrote:
> Serves me right for trying to do this quickly using the GH UI,

Well, the obvious advantage of that is the immediate visual control of the ewndered document.

> now I need to figure out how to edit a commit. :-)

I don't know if it is possible in the UI at all.


Of course you know this, but you can do it locally with `git commit --amend`

Reply:
One alternative: If the change request is added as a suggested diff, you can apply it as an additional commitfrom the GH UI, but then it would need to be squashed with your original commit and I think that is not possible from the UI but has to be done locally with `git rebase -i` 

So as a summary, I think it is not possible yo do it entirely from the UI, only partially.

Reply:
@phlogistonjohn , we could also fix this in a follow-up PR :)

Reply:
see #352